### PR TITLE
syncbackfree: Add version 9.2.30.0

### PR DESCRIPTION
**Pull request by issaclin32**
**SyncBackFree** ([homepage](https://www.2brightsparks.com/download-syncbackfree.html)) is a data backup and synchronize software.

Notes:

* *SyncBackPro* ([syncbackpro.json](https://github.com/lukesampson/scoop-extras/blob/master/bucket/syncbackpro.json)) has 32-bit and 64-bit versions, but **SyncBackFree** has only one application for both 32-bit and 64-bit.
> Windows 10, Windows 8, Windows 7 and Windows Vista. Both 32-bit and 64-bit versions of Windows are supported.

* **persist**: *SyncBackFree* does not provide the option for **custom backup profile location** like *SyncBackPro* does. Therefore the method used in [SyncBackPro package](https://github.com/lukesampson/scoop-extras/blob/master/bucket/syncbackpro.json) cannot be used here. I tried to put the content below in `settingsFolder.ini`, but it did not work for *SyncBackFree*.
(*SyncBackFree* stores its backup profiles in `$Env:LocalAppData\2BrightSparks\SyncBackFree` by default)
```
[Settings]
Folder=%THISPATH%settings
Restricted=1
```

### CODE REVIEW

**Comment by issaclin32** [comment]
> How do you install it? There is no extraction defined or installer

I forgot to add `innosetup: true`. Sorry.

**Comment by Ash258** [comment]
Why do you open PR when you never installed/uninstalled/updated/checkver the manifest?

File: bucket/syncbackfree.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "9.2.12.0",
+    "description": "Data backup and synchronize software",
```
Comment (Ash258) [inline bucket/syncbackfree.json]:
```suggestion
    "description": "Data backup and synchronization software",
```

File: bucket/syncbackfree.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "homepage": "https://www.2brightsparks.com/download-syncbackfree.html",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.2brightsparks.com/download-syncbackfree.html#tab2"
+    },
+    "url": "https://www.2brightsparks.com/assets/software/SyncBack_Setup.exe",
+    "hash": "4995b24b6a9c2b3e71365a967b469220bfbe4f4bd42ddd4133714f1938a0a6f8",
```
Comment (Ash258) [inline bucket/syncbackfree.json]: Latest version in download page is `9.2.30.0` 
Hash in this manifest is for version 9.2.12.0, but this hash is same for 30

image

**Pull request merged**